cob_bringup/components/ros_control_base.launch

```diff
@@ -25,10 +26,12 @@
 		<arg name="reset_errors_before_recover" value="$(arg reset_errors_before_recover)"/>
 	</include>
 
-	<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
-		<arg name="timeout" default="2.0"/>
-		<arg name="threshold" default="0.174533"/>
-	</include>
+	<group unless="$(arg disable_stuck_detector)">
```

Comment:
you can adjust the parameters 
```
<arg name="timeout" default="2.0"/>
<arg name="threshold" default="0.174533"/>
```

if you set the time to 4sec and the threshold to 0.5 you shouldn't see the stuck detector to get active. If you still see the stuck detector to stop the base there's someting else wrong in the setup (probably controller tuning on elmos).